Description: Split original patch by Slawomir and update the same to apply
neatly.
Author: Bhavani Shankar <bhavi@ubuntu.com>


--- cuneiform-1.1.0+dfsg.orig/cuneiform_src/Kern/rstr/src/acc_tabs.c
+++ cuneiform-1.1.0+dfsg/cuneiform_src/Kern/rstr/src/acc_tabs.c
@@ -2821,8 +2821,8 @@ if( CodePages[language]==CSTR_EASTEUROPE
     strcpy(decode_ASCII_to_[(uchar)liga_j      ],   "_j_");
     strcpy(decode_ASCII_to_[(uchar)liga_exm    ],   "_!_");
     strcpy(decode_ASCII_to_[(uchar)liga_qm     ],   "_?_");
-    strcpy(decode_ASCII_to_[(uchar)liga_inv_exm],   "_!!_");
-    strcpy(decode_ASCII_to_[(uchar)liga_inv_qm ],   "_??_");
+    strcpy(decode_ASCII_to_[(uchar)liga_inv_exm],   "_!_");
+    strcpy(decode_ASCII_to_[(uchar)liga_inv_qm ],   "_?_");
     strcpy(decode_ASCII_to_[(uchar)liga_bull   ],   "\xB0");
     strcpy(decode_ASCII_to_[(uchar)left_quocket],   "«");
     strcpy(decode_ASCII_to_[(uchar)right_quocket],  "»");
